localisation/en_dtac/strings.json

```diff
@@ -1,10 +0,0 @@
-{
-  "LABEL_REVIEW_APPROVE": "Mark as reviewed",
-  "LABEL_REVIEW_RESSUBMIT": "Request clarification",
-  "LABEL_REVIEW_DECICION_CONFORM": "Reviewed",
-  "LABEL_REVIEW_DECISION_NON_CONFORM": "Clarification required",
-  "DECISION_CONFORM": "Move application to the next stage",
-  "DECISION_NON_CONFORM": "Mark application as unsuccessful",
-  "OUTCOME_APPROVED": "Granted!",
-  "OUTCOME_REJECTED": "Unsuccessful this time"
-}
```

Comment:
As mentioned `/localisation` removed from repo and is restored locally when starting up or loading a snapshot. This keeps it behaving the same as "files" and "snapshots".

Reply:
Oh... Good point. I have added a change on the keys to be in core_templates also - along with copying latest version of lcoalisation in pt_an to be there also - straight commited to develop